### PR TITLE
[Bug] [streamx-console] Fix modify user and set mobile to empty will alert mobile手机号不合法.

### PR DESCRIPTION
**Pull request by GOODBOY008**
Link issue https://github.com/streamxhub/streamx/issues/1415 .
Fix modify user and set mobile to empty will alert `mobile手机号不合法`. Translate front-end alert message to english.

### CODE REVIEW

**Comment by wolfboys** [comment]
It may be a global project in the future. Only the mobile phone number rules of China are considered here. Whether this mobile phone number is required can be discussed later. If it is not necessary, this field can be removed

File: streamx-console/streamx-console-service/src/assembly/script/final.sql [hunk 1]
```diff
@@ -572,7 +571,7 @@ UNIQUE KEY `UN_USERNAME` (`NICK_NAME`) USING BTREE
 -- Records of t_user
 -- ----------------------------
 BEGIN;
-INSERT INTO `t_user` VALUES (100000, 'admin', '', 'ats6sdxdqf8vsqjtz0utj461wr', '829b009a6b9cc8ea486a4abbc38e56529f3c6f4c9c6fcd3604b41b1d6eca1a57', 'benjobs@qq.com', '13800000000', '1', NOW(), NULL,NULL,NULL,NULL,NULL );
+INSERT INTO `t_user` VALUES (100000, 'admin', '', 'ats6sdxdqf8vsqjtz0utj461wr', '829b009a6b9cc8ea486a4abbc38e56529f3c6f4c9c6fcd3604b41b1d6eca1a57', 'benjobs@qq.com', '1', NOW(), NULL,NULL,0,NULL,NULL );
```
Comment (wolfboys) [inline streamx-console/streamx-console-service/src/assembly/script/final.sql]: Oh, My God, my private email has been exposed.😫

Comment (GOODBOY008) [reply streamx-console/streamx-console-service/src/assembly/script/final.sql]: Ok，I will remove set  field `EMAIL` to `NULL`.

**Comment by wolfboys** [comment]
LGTM

**Pull request merged**